src/main/java/com/fasterxml/jackson/databind/deser/BeanDeserializerBase.java

```diff
@@ -1735,18 +1735,41 @@ protected void handleIgnoredProperty(JsonParser p, DeserializationContext ctxt,
      * @param p (optional) If not null, parser that has more properties to handle
      *   (in addition to buffered properties); if null, all properties are passed
      *   in buffer
+     * @deprecated use {@link #handlePolymorphic(JsonParser, DeserializationContext, StreamReadConstraints, Object, TokenBuffer)}
      */
+    @Deprecated
```

Comment:
Is there any case where `StreamReadConstraints` are not accessed from passed-in `JsonParser` (except if that is `null` in which case we use defaults)? If not, I think I'd prefer leaving signatures as-is and just getting constraints from parser.

Reply:
but the defaults could be wrong for the user's case, they might want bigger limits - why not try to pass on the streamReadConstraints from the parent parser. This only happens when handlePolymorphic is called with a null parser instance but the calling code has access to a parent instance of the parser.

Reply:
I think I phrased my question wrong: I do not see why we need a new overload, instead of just accessing constraints from passed-in `JsonParser`, whichever that is?

Reply:
Oh. Now I see why: there is that one case of `null` being handed as parser. But I'll need to see why it's not passing actual parser....

Reply:
and ok. Right. `null` is passed if nothing is to be read from it. Got it.